Assets/Scripts/TitleManagerScript.cs

```diff
@@ -68,20 +75,22 @@ private void Awake()
         DataManagerScript.isChallengeMode = false;
         DataManagerScript.isSinglePlayerMode = false;
         DataManagerScript.arenaType = -99;
+        
+		highlight = GameObject.Find("Highlight");
+
         // Init controller maps
         for (int i = 0; i < 4; i++) {
             gamepads[i] = new JoystickButtons(i + 1);
         }
 
         if (DataManagerScript.isFirstPlay)
         {
-            Invoke("AllowInput", 9f);
+            Invoke("AllowInput", 4f);
```

Comment:
Does this allow a user to 'slide up' the logo before the video is complete? Because if so there might be an issue where the 'white flash' of the video doesn't fill up the whole screen

Reply:
Ooh nm, I see it is timed to avoid that. I like it -

Reply:
Oh yeah, meant to ask you if you liked this! Did indeed time it with the flash. I originally set it to just be skippable for testing but then went down a rabbit whole of making it "nice".